manifest.php

```diff
@@ -35,11 +39,11 @@
     'label' => 'LTI library',
     'description' => 'TAO LTI library and helpers',
     'license' => 'GPL-2.0',
-    'version' => '11.12.0',
+    'version' => '11.17.0',
```

Comment:
i think we used a bit too many version ,  maybe we can reduce the big jump because currently we are taking this as a whole development

Reply:
Yes, we should not have updated the version inside of the integration branch till now. I vote to fix it too.